### DISCUSSION TITLE
8 Kanal 16A Relaismodul von haus-bus.de in Verbindung mit CCU2 und alter HM-wired -- funktioniert das in HA?

### DISCUSSION THREAD

**Discussion by 17sasa** [SukramJ/aiohomematic#1662] [Q&A]
Hallo zusammen,

ich habe vor mir so ein  8 Kanal 16A Relaismodul von haus-bus.de zuzulegen.

Nun bin ich auf HA umgestiegen und frage mich, ob das Modul dann mit HA funktioniert, bzw. auf was ich ggf. achten muss?

Mit den Jungs von Haus-Bus habe ich im Vorfeld telefoniert (nicht aber wegen HA gefragt).
Sehr angenehmes Gespräch, da die dort generell "etwas bewegen wollen", und dabei wurde mir auch die offizielle Freigabe zur CCU2 mitgeteilt.

---------------------------
hier mal die Infos dazu...

 
   
  | 8 Relaisausgänge a 16A (Schaltspannung: 277 VAC / 30 VDC)
 
  | Jedes Relais hat 3 Anschlüsse (Zuleitung, Öffner- und Schließerkontakt)
  | 8 Digitaleingänge activeHigh bis 30V
  | RFID Leser anschließbar (Infos)
  | Spannungsversorgung 7-30V
  | RS485 Bus zum Betrieb an einer RS485 Extension oder an einem beliebigen Haus-Bus Modul mit LAN Interface
  | Optionale Deckelplatine zur manuellen Steuerung aller Kanäle und LED Statusanzeige
  | Optische Statusanzeige: Status-LEDs (1 x rot, 1 x grün)
  | ABS-Hutschienengehäuse 107 mm x 90 mm x 60 mm (B x T x H)
  | Maximale Stromaufnahme: 280 mA
  | Detailierte Stromaufnahme: 40 mA Grundlast + 30 mA pro aktivem Relais
  | Download: Produktbeschreibung als PDF

 http://www.haus-bus.de/?showProduct=7

--------------------------------

Vielen Dank und einen schoenen Sonntag
VG
sasa

**Comment by SukramJ** [comment]
Keine Ahnung. Anstecken und schauen was passiert. 
Da hier ein generischer Ansatz verfolgt wird sollte das Gerät erkannt und die Basisfunktionen sollten bereitstehen.
Etwaige weitere Anpassungen kannst Du in HA selber über Templates machen.

**Reply by SukramJ** [reply]
Ich übernehme natürlich keine Funktionsgarantie.

**Comment by 17sasa** [comment]
Hört sich ja erstmal nicht "soo negativ" an. Da wurde mir im Vorfeld dann doch mehr bange gemacht.

Wie verhält es sich denn der sogenannten "Wiegand Protokoll Schnittstelle" , diese gibt es ja auch als eigenes Bauteil vom Homematic. 
_An den Haus-Bus Modulen können beliegige RFID Leser mit dem Wiegand Protokoll (26 oder 34 Bit) betrieben werden_
Sollte das dann ebenfalls funktionieren?

Danke
VG

**Reply by SukramJ** [reply]
Frag jetzt nicht nach jedem Gerät. Das oben gesagte gilt für alle Geräte.
